apps/settings/js/panels/addons/addons_list.js

```diff
@@ -34,13 +35,21 @@ define(function(require) {
       }
     },
 
+    _sortByName: function(addons) {
+      return addons.sort((a, b) => {
+        return a._app.manifest.name.localeCompare(b._app.manifest.name);
+      });
+    },
+
     _update: function() {
       return this._filter ?
         Promise.all(this._addonManager.addons.array.map(addon =>
           this._addonManager.addonAffectsApp(addon, this._filter).then(
             affects => affects ? addon : undefined))).then(
-              addons => this._addons.reset(addons.filter(addon => addon))) :
-        Promise.resolve(this._addons.reset(this._addonManager.addons.array));
+              addons => this._addons.reset(this._sortByName(addons.filter(
+                addon => addon)))) :
```

Comment:
It looks more eye friendly to break line at the same end `this._sortByName(`
addons.filter(addon => addon)))) : :)